webform.go

```diff
@@ -97,13 +97,13 @@ func WebFormContents(invokeURI, metadataURI string, descs []*desc.MethodDescript
 // that first verifies the CSRF token header before delegating to a
 // RPCInvokeHandler.
 func WebFormScript() []byte {
-	return webform.WebFormScript()
+	return webform.MustAsset("webform.js")
```

Comment:
Per other comment, we should revert this change and just hand-write `WebFormScript` and company if necessary. This package should not need to know internal details like these file names.